include/level_commands.h

```diff
@@ -449,6 +449,10 @@ enum GoddardScene {
 #define SET_ECHO(console, emulator) \
     CMD_BBBB(LEVEL_CMD_SET_ECHO, 0x04, console, emulator)
 
+// Required for backwards compatibility with Fast64
+#define MACRO_OBJECTS(objList) \
```

Comment:
Does this need to NOP? I feel like it could just be nothing tbh.

Reply:
the level command has a comma after it so the array wouldnt compile

Reply:
Blah